### DISCUSSION TITLE
Dealing with Archive/Trash items

### DISCUSSION THREAD

**Discussion by J-J-E** [TheLastGimbus/GooglePhotosTakeoutHelper#315] [I NEED HELP]
Hello everyone. First off, this is an amazing tool and thank you for making it! I am trying to deal with 400gb of photo/video from 2007-current. I want to extract the data into the year/month folder structure, ignoring album relationships, sym links, etc. However, I want the contents of my archive and my trash to also appear in the output ALL_PHOTOS folder. 

What options should I select for this? I can't seem to get the contents of the archive/trash to be copied over when I select option 3:

[3] nothing: Just ignore them and put year-photos into one folder. WARNING: This ignores Archive/Trash !!!

**Comment by TheLastGimbus** [comment]
Looks like `json` option would be best for you :ok_hand:
